SetupDataPkg/Tools/GenCfgData.py

```diff
@@ -1445,6 +1454,7 @@ def _update_tree(exec, bin_data):
 
         # update the exec itself
         exec = self.locate_exec_from_tag(tag_id)
+        # print (f"bin_data: {bin_data} exec: {exec} tag_id: {tag_id}")
```

Comment:
did you mean to leave this is? If so consider a global DEBUG_PRINTS instead of comments

Reply:
Nope, this is junk I should get rid of.

Reply:
Removed.